stylix/testbed/default.nix

```
@@ -33,9 +33,15 @@ let
               )
             )
             {
-              inherit (inputs.nixvim.nixosModules) nixvim;
               inherit (inputs.spicetify-nix.nixosModules) spicetify;
 
+              nixvim-integrated = inputs.nixvim.nixosModules.nixvim;
+
+              nixvim-standalone = {
+                config.lib.stylix.testbed.makeNixvim =
+                  inputs.nixvim.legacyPackages.${pkgs.stdenv.hostPlatform.system}.makeNixvim;
+              };
```

Sorry for not spotting this nitpick before:

```suggestion
              nixvim-standalone.config.lib.stylix.testbed.makeNixvim =
                inputs.nixvim.legacyPackages.${pkgs.stdenv.hostPlatform.system}.makeNixvim;
```

If we do this, then we should also the same for the `zen-browser` declaration below: https://github.com/nix-community/stylix/pull/1906.